packages/clay-charts/src/Geomap.js

```diff
@@ -72,6 +74,10 @@ class Geomap extends Component {
 			return;
 		}
 
+		if (this._pollingInterval) {
+			clearTimeout(this._pollingInterval);
+		}
```

Comment:
This should go to DataComponent, as ChartBase has no knowledge at all about polling